testsuite/setup.py

```diff
@@ -128,8 +128,9 @@ def dynamic_author_list():
 if __name__ == '__main__':
     try:
         dynamic_author_list()
-    except (OSError, IOError):
-        warnings.warn('Cannot write the list of authors.')
+    except (OSError, IOError) as e:
```

Comment:
remove creation of `MDAnalysisTests/authors.py` and remove the code that imports `authors.py` in `__init__.py`.